### PR TITLE
[WIP] OSDOCS-7755 Adding documentation links to priority ROSA notifications

### PR DESCRIPTION
**Pull request by laubai**
### What type of PR is this?
documentation

### What this PR does / why we need it?
MOBB-F2F: adding documentation URLs and/or more detail for DiskPressure and NetworkUnavailable notifications

### Which Jira/Github issue(s) this PR fixes?
Partial fix for
https://issues.redhat.com/browse/OSDOCS-7755
https://issues.redhat.com/browse/OSDOCS-7756

_Fixes #_

### Special notes for your reviewer:
MOBB-F2F

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Comment by laubai** [comment]
CC @jewzaam @tsze-redhat for review

File: hack/00-osd-managed-cluster-config-production.yaml.tmpl [hunk 1]
```diff
@@ -22104,7 +22104,7 @@ objects:
         - name: NodeConditionDiskPressureNotification
           severity: Warning
           summary: Worker node is experiencing DiskPressure
-          activeBody: DiskPressure condition is true for at least one worker.
+          activeBody: DiskPressure condition is true for at least one worker. Consider moving application storage off the worker node and into persistent volumes. For more information, see (https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/storage/configuring-persistent-storage)[Configuring persistent storage].
```
Comment (jewzaam) [inline hack/00-osd-managed-cluster-config-production.yaml.tmpl:22107]:
```suggestion
          activeBody: DiskPressure condition is true for at least one worker. Consider moving application storage off the worker node and into persistent volumes. For more information, see [Configuring persistent storage](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/storage/configuring-persistent-storage).
```

File: hack/00-osd-managed-cluster-config-production.yaml.tmpl [hunk 2]
```diff
@@ -22122,7 +22122,7 @@ objects:
         - name: NodeConditionNetworkUnavailableNotification
           severity: Warning
           summary: Worker node is experiencing NetworkUnavailable
-          activeBody: NetworkUnavailable condition is true for at least one worker.
+          activeBody: NetworkUnavailable condition is true for at least one worker. Red Hat SRE will remedy if not automatically resolved.
```
Comment (jewzaam) [inline hack/00-osd-managed-cluster-config-production.yaml.tmpl:22125]:
```suggestion
          activeBody: NetworkUnavailable condition is true for at least one worker. Red Hat SRE will be alerted and triage issues.  If action is required, a proactive case will be opened by Red Hat.
```

I don't want to say we will resolve it, after reflecting there is a class of rare change customers can do to break networks.  We shouldn't imply it's Red Hat's responsibility in all cases.

**Review by jewzaam — CHANGES_REQUESTED** [review]
I would like PM review before we merge.

**Pull request closed without merging**